kotlinx.fuzz.gradle/src/main/kotlin/kotlinx/fuzz/gradle/KFuzzPlugin.kt

```diff
@@ -30,6 +34,14 @@ abstract class KFuzzPlugin : Plugin<Project> {
             add("testRuntimeOnly", "org.jetbrains:kotlinx.fuzz.gradle:$pluginVersion")
         }
 
+        val projectPropertiesMap: Map<String, String> = project.properties.mapValues { (_, v) -> v.toString() }
+        val fuzzConfigDSL = project.extensions.create<FuzzConfigDSL>(
+            "fuzzConfig",
+            FuzzConfigDSL::class,
```

Comment:
You can remove this line. `kotlin-dsl` containtains extention function without this parameter